_posts/0000-01-02-ashok1801.md

```diff
@@ -0,0 +1,6 @@
+---
+layout : slide
+title : "Welcome to our second slide"
+---
+This is first slide
+
```

Comment:
This is Second File 
Next One will be Third File
Next will be Fourth File
Then will get Fifth file
Last One will be Sixth File